client/src/components/ActionMenu/ActionMenu.js

```diff
@@ -52,7 +54,9 @@ class ActionMenu extends PureComponent {
         toggle={this.toggle}
         {...restProps}
       >
-        <DropdownToggle className={toggleClassName} aria-label={i18n._t('Admin.ACTIONS', 'View actions')} {...dropdownToggleProps} />
+        <DropdownToggle className={toggleClassName} title={toggleText} aria-label={toggleText} {...dropdownToggleProps} >
```

Comment:
Adding a `title` attribute here because:
1. There used to be a `.visually-hidden` text that isn't there anymore. That text was used in behat to identify buttons - now the `title` attribute is used instead
2. This gives sighted users the same context that screen reader users have.